src/NavBar/NavBar.js

```diff
@@ -16,7 +19,10 @@ export function NavBar(props) {
               id="searh-bar"/>
             </label>
           </div>
+          <Link to='/login'>
           <button onClick={props.logIn} className="Login-button">Log in</button>
+          </Link>
```

Comment:
I think I left this spot open in the tests because I knew it would have to change after implementing Router. This is just a comment to remind me to go back and update that piece.